nixos/modules/services/networking/bind.nix

```diff
@@ -211,25 +212,33 @@ in
 
     networking.resolvconf.useLocalResolver = mkDefault true;
 
-    users.users.${bindUser} =
-      {
+    users = {
+      groups = {
+        "${bindGroup}" = {
+          gid = config.ids.gids.bind;
+        };
+      }
+      users.${bindUser} = {
         uid = config.ids.uids.bind;
+        groups = "${bindGroup}";
         description = "BIND daemon user";
       };
+    };
 
     systemd.services.bind = {
       description = "BIND Domain Name Server";
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
 
       preStart = ''
-        mkdir -m 0755 -p /etc/bind
+        mkdir -m 7600 -p /etc/bind
```

Comment:
setuid, setgid, sticky bit
owner may read/write

What is the idea here?

Reply:
Group having rights for `rw` and user having `rwx` others nothing.

The idea being that nothing here needs executing, but keeping that open for the bind user in case people are doing scripts etc..

Reply:
I think you have misconceptions about chmod calculation, because 7600 does not do what you say your idea was. 

Also I'm not sure what you think handing out write permissions will accomplish, because likely there are only symlinks into the nix store within /etc/bind.

Reply:
You are correct it should be `760` thanks for catching it

Reply:
about the `/etc/bind` afaik that location is mutable so it makes sense for runtime maintenance